src/reducers/application.js

```diff
@@ -43,18 +43,22 @@ export default (state = initialState, action) => {
       const messages = [...state.messages];
       messages.push({
         message: action.payload.message,
-        type: action.payload.type,
+        messageInterface: action.payload.messageInterface,
+        messageSeverity: action.payload.messageSeverity,
         key: Date.now() + Math.random(),
+        open: true,
       });
       return {
         ...state,
         messages,
       };
     }
     case CLEAR_MESSAGE: {
+      const messages = [...state.messages];
```

Comment:
Should we support clearing all messages somehow?

Reply:
Sure! The next commit (which I just pushed) has that as a feature, and I also removed the now-unnecessary stuff from my original commit (actions/reducers for snackbar).